tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
```

Comment:
in general I'm not sure we even used tox with ccm, and I don't know if this tox.ini is working
I think we can remove it

Reply:
yup. also, nose is practically dead. but i'd prefer removing this in a separate PR. as this change is orthogonal to a more general cleanup which should drop unused testing harness. what do you think?

Reply:
o.k. no rush

Reply:
#446